tasks/clear_abi.js

```diff
@@ -58,6 +60,14 @@ subtask(
     }
 
     await fs.promises.rm(file);
+    if (configForPath.tsWrapper) {
```

Comment:
Should attempt to delete the TS file regardless of whether the config option is currently set.  The user may have turned of the config option because they no longer need the TS files.

Reply:
Oaky, the typescript compiler doesn't work this way (it always leaves files after you disable configs that generate them), but I now see that Hardhat took a different approach so it makes sense to delete them here to work well with the expected Hardhat behavior 👍